ray-operator/test/e2e/rayjob_lightweight_test.go

```diff
@@ -76,12 +82,18 @@ env_vars:
 			WithSpec(rayv1ac.RayJobSpec().
 				WithSubmissionMode(rayv1.HTTPMode).
 				WithEntrypoint("python /home/ray/jobs/fail.py").
+				WithEntrypointNumCpus(2).
```

Comment:
How about simply updating "Successful RayJob" and leave other tests the same?

Reply:
Sure, no problem!